### PR TITLE
Updated area

### PR DESCRIPTION
**Pull request by jrajak10**

### CODE REVIEW

File: docs/osindex.html [hunk 1]
```diff
@@ -42,6 +42,7 @@
     <div class='map-overlay-inner'>
         <button id='request'>Find localBuildings</button>
         <div>Total features: <span id='feature-count'></span></div>
+        <div>Total area <b>(metres squared)</b>: <span id='area'></span></div>
```
Comment (sebastianovide) [inline docs/osindex.html]: typo: https://en.wikipedia.org/wiki/Square_metre

Comment (sebastianovide) [inline docs/osindex.html]: also, what is total area ? (perhaps, "total buildings area" ?)

File: docs/osindex.html [hunk 2]
```diff
@@ -178,6 +179,7 @@
 
         map.getSource('localBuildings').setData(geojson);
         document.getElementById('feature-count').innerHTML = '';
+        document.getElementById('area').innerHTML = 0;
```
Comment (sebastianovide) [inline docs/osindex.html]: it shows 0 for a brief.... perhaps leave it empty until when you have the number.

File: docs/osindex.html (hunk 1 above)
Comment (sebastianovide) [reply docs/osindex.html]: also, should you use a number more human friendly ? (491138.964 not very friendly)

Comment (jrajak10) [reply docs/osindex.html]: Okay. What would you suggest? Would it be something like rounding to nearest whole number, and adding commas to every thousand, such as 491,138? Or even just rounding it to the nearest thousand squared metres?

Comment (jrajak10) [reply docs/osindex.html]: So it returns just 491

File: docs/osindex.html (hunk 2 above)
Comment (jrajak10) [reply docs/osindex.html]: Made this change

File: docs/osindex.html (hunk 1 above)
Comment (jrajak10) [reply docs/osindex.html]: Added a sup tag to the 2 so it reads m^2

Comment (sebastianovide) [reply docs/osindex.html]: Change unit ? Did u ask Rob?

Comment (jrajak10) [reply docs/osindex.html]: I'll ask him

**Pull request merged**